web/plugin-manager/src/helpers/alertHelpers.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+function alertHelper(typeAlert: string, message: string, remove: () => void) {
+    const successAlert = (
```

Comment:
Why the const are defined in the function? This means that you will have 3 different const set each time you call the alerthelper. Don't you think this should be defined outside of the function, hence being defined only once ?

Reply:
Changed in functions to avoid the memory allocation. But it's not significant in term of gains